Source/GSXib5KeyedUnarchiver.m

```diff
@@ -207,6 +225,8 @@ + (void) initialize
                             @"NSWindowTemplate", @"window",
                             @"NSView", @"tableCellView",
                             @"IBUserDefinedRuntimeAttribute5", @"userDefinedRuntimeAttribute",
+                            @"GSStoryboardScene", @"scene",
```

Comment:
Why would we want the create GNUstep specific objects here?????

Reply:
There don't need to be any modifications to the XIB loading mechanism at this point, I am going to take these out as well as the modifications to handle the GS prefix.